packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1212,7 +1215,7 @@ class PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
       icon: widget.icon ?? Icon(Icons.adaptive.more),
       padding: widget.padding,
       splashRadius: widget.splashRadius,
-      iconSize: widget.iconSize ?? 24.0,
+      iconSize: widget.iconSize ?? iconTheme.size ?? 24.0,
```

Comment:
Can we make the "24.0" into a private constant at the top of this file?  e.g. `_kDefaultIconSize`

Reply:
Updated test to check default icon size, theme icon size, and icon size parameter, also added comments in the test.